src/components/IconButtonBar/IconButtonBar.js

```diff
@@ -41,14 +41,25 @@ const IconButtonBar = ({
     }
   );
 
-  const renderIconButton = action => (
-    <IconButton
-      {...action}
-      key={action.label || `${namespace}__button--icon`}
-      size={size}
-      tooltip={tooltip}
-      tooltipDirection={iconTooltipDirection}
-    />
+  const renderIconButton = (action, index) => (
+    <Fragment key={action.label || `${namespace}__button--icon--${index}`}>
+      {(action.divider === 'left' || action.divider === 'sides') && (
+        <span className={`${namespace}__divider`} aria-hidden>
+          <span className={`${namespace}__divider__inner`} aria-hidden />
```

Comment:
Awesome! As I understand it, the `aria-hidden` declaration on the parent `divider` element should be sufficient to remove this from the accessibility tree.

... Unless different screen readers handle this in different ways? What's the rationale for adding it to the `inner` element?